sixtyfps_compiler/generator/cpp.rs

```diff
@@ -599,8 +599,13 @@ pub fn generate(doc: &Document, diag: &mut BuildDiagnostics) -> Option<impl std:
 
     file.declarations.extend(doc.root_component.embedded_file_resources.borrow().iter().map(
         |(path, id)| {
-            let data =
-                std::fs::read(path).expect(&format!("unable to read file for embedding: {}", path));
+            let data = match path.strip_prefix("builtin:/") {
+                None => std::fs::read(path)
+                    .expect(&format!("unable to read file for embedding: {}", path)),
```

Comment:
Pre existing: this should not be a panic, but a compilation error.
Or is it already handled before?
Well, in that case, it should be a string literal or a unwrap_or with a panic so there is no format in the non error case